odata-url-conventions/Appendix.md

```diff
@@ -62,12 +62,6 @@ https://www.ecma-international.org/publications-and-standards/standards/ecma-262
 
 -------
 
-# Appendix ##asec Safety, Security and Privacy Considerations
```

Comment:
Better move the section heading into the TODO comment, keep the comment and remove/replace it when applying https://github.com/oasis-tcs/odata-specs/issues/378